modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/internal/js/loader/modules/extender/npm/NPMResolverProvider.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.data.set.taglib.internal.js.loader.modules.extender.npm;
+
+import com.liferay.frontend.js.loader.modules.extender.npm.NPMResolver;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Chema Balsas
+ */
+@Component(immediate = true, service = {})
```

Comment:
So I have a doubt here about the `@author` annotation: I'm guessing you left Chema's name because this class is based on something very similar a while back - if that's the case, then I don't see a problem and you can safely ignore this comment, but otherwise, you might want to put your name instead.

Reply:
Yeah, it's a complete copy of https://github.com/liferay-frontend/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/internal/js/loader/modules/extender/npm/NPMResolverProvider.java

Reply:
Purfect